src/core/addonManager.js

```diff
@@ -100,6 +102,26 @@ export function getAddon(
   return Promise.reject(new Error('Cannot check add-on status'));
 }
 
+export function hasAbuseReportPanelEnabled({
+  navigator,
+}: { navigator: PrivilegedNavigatorType } = {}) {
+  if (hasAddonManager({ navigator })) {
+    const _navigator = navigator || window.navigator;
+    return _navigator.mozAddonManager.abuseReportPanelEnabled;
+  }
+  return undefined;
```

Comment:
Shouldn't we return `true` or `false`? Or do we need `undefined` to know that `mozAddonManager` is not available?

Reply:
Agreed - we can just return `true` or `false`.